### PR TITLE
Update ZIpline and Basejump code review instructions, remove gist export

### LINKED ISSUE #4699: Include CodeReview Room as possible  Feedback area
Right now the only "official" way to get feedback on a project is to tweet it out.  This won't scale.  We should encourage campers too use the CodeReview Room. 

Should be included in the Zipline into and getting set sections.  Elsewhere?

### PR DESCRIPTION
**Pull request by QuincyLarson**
Closes #4699

### CODE REVIEW

File: server/views/coursewares/showBonfire.jade [hunk 1]
```diff
@@ -73,11 +73,9 @@ block content
...
-                                  i.fa.fa-twitter &thinsp;
-                                      = phrase
+                          a.btn.btn-lg.btn-block.btn-twitter(target="_blank", href="https://twitter.com/intent/tweet?text=I%20just%20#{verb}%20%40FreeCodeCamp%20#{name}&url=http%3A%2F%2Ffreecodecamp.com/challenges/#{dashedName}&hashtags=LearnToCode, JavaScript", onclick="ga('send', 'event', 'twitter', 'share', 'challenge completion share');")
+                              i.fa.fa-twitter &thinsp;
+                                  = phrase
                       else
                           a#next-challenge.btn.btn-lg.btn-primary.btn-block(href="/challenges/next-challenge?id="+challengeId) Go to my next challenge (ctrl + enter)
                           a.btn.btn-lg.btn-danger.btn-block#gist-share Share your solution as a GitHub gist
```
Comment (bugron) [inline server/views/coursewares/showBonfire.jade]: @QuincyLarson there is also another Gist button.

**Comment by QuincyLarson** [comment]
Note - I messed up on the rebasing and as a result was not able to practically squash my commits without also overwriting other people's commits.

**Comment by SaintPeter** [comment]
Look good.

**Comment by adamisntdead** [comment]
Just wondering - as I am working on a new gist feature - why was this removed before?

**Pull request merged**